source/tyr/tyr/tasks.py

```diff
@@ -276,3 +276,26 @@ def heartbeat():
             producer = connection.Producer(exchange=exchange)
             producer.publish(task.SerializeToString(), routing_key='{}.task.heartbeat'.format(instance.name))
 
+
+@celery.task()
+def create_autocomplete_depot(name):
+    autocomplete_path = current_app.config['TYR_AUTOCOMPLETE_DIR']
+
+    if os.path.exists(autocomplete_path):
+        name_path = os.path.join(autocomplete_path, name)
```

Comment:
can you abstract this a bit ?

maybe put in AutocompleteParameter a `source_dir` / `backup_dir` method ?